README.md

```diff
@@ -6,6 +6,23 @@ She's Coding Project is part of [She's Coding slack community](https://forms.gle
 
 Please find an installation guide for your environment in our [Wiki](https://github.com/shescoding/projects-platform-frontend/wiki).
 
+## 🏃 Running the project
```

Comment:
Heya, our apologies, this file has been rewritten here since after the GHC https://github.com/shescoding/projects-platform-frontend/wiki/Project-Installation-Guide, plz remove changes to Readme.md